ios/FluentUI.Demo/FluentUIDemoTests/TableViewCellTest.swift

```diff
@@ -8,8 +8,93 @@ import XCTest
+        XCTAssertEqual(app.cells.element(boundBy: 14).identifier, "Table View Cell with title \(title) and subtitle \(subtitle), with a subtitle leading image")
+    }
+
+    func testTripleLine() throws {
+        XCTAssertEqual(app.cells.element(boundBy: 15).identifier, "Table View Cell with title \(title), subtitle \(subtitle), and footer \(footer), with a leading image")
+        XCTAssertEqual(app.cells.element(boundBy: 16).identifier, "Table View Cell with title \(title), subtitle \(subtitle), and footer \(footer), with a leading image and a chevron")
+        XCTAssertEqual(app.cells.element(boundBy: 17).identifier, "Table View Cell with title \(title), subtitle \(subtitle), and footer \(footer), with a leading image and a details button")
+        XCTAssertEqual(app.cells.element(boundBy: 18).identifier, "Table View Cell with title \(title), subtitle \(subtitle), and footer \(footer), with a leading image and a checkmark")
+        XCTAssertEqual(app.cells.element(boundBy: 19).identifier, "Table View Cell with title \(title), subtitle \(subtitle), and footer, with a leading image, a subtitle trailing image, and a footer trailing image")
+    }
+
+    func testCustomView() throws {
+        XCTAssertEqual(app.cells.element(boundBy: 25).identifier, "Table View Cell with title \(customTitle)")
+        XCTAssertEqual(app.cells.element(boundBy: 26).identifier, "Table View Cell with title \(customTitle), with a chevron")
+        XCTAssertEqual(app.cells.element(boundBy: 27).identifier, "Table View Cell with title \(customTitle), with a details button")
+        XCTAssertEqual(app.cells.element(boundBy: 28).identifier, "Table View Cell with title \(customTitle), with a checkmark")
+        XCTAssertEqual(app.cells.element(boundBy: 29).identifier, "Table View Cell with title \(customTitle)")
+    }
+
+    func testLongLabels() throws {
+        XCTAssertEqual(app.cells.element(boundBy: 30).identifier, "Table View Cell with title \(longTitle), subtitle \(longSubtitle), and footer \(longFooter), with a leading image")
+        XCTAssertEqual(app.cells.element(boundBy: 31).identifier, "Table View Cell with title \(longTitle), subtitle \(longSubtitle), and footer \(longFooter), with a leading image and a chevron")
+        XCTAssertEqual(app.cells.element(boundBy: 32).identifier, "Table View Cell with title \(longTitle), subtitle \(longSubtitle), and footer \(longFooter), with a leading image and a details button")
+        XCTAssertEqual(app.cells.element(boundBy: 33).identifier, "Table View Cell with title \(longTitle), subtitle \(longSubtitle), and footer \(longFooter), with a leading image and a checkmark")
+        XCTAssertEqual(app.cells.element(boundBy: 34).identifier, "Table View Cell with title \(longTitle), subtitle \(longSubtitle), and footer \(longFooter), with a leading image, a title trailing image, a subtitle trailing image, and a footer trailing image")
+    }
+
+    // ensures that selection/deselection adds/removes unread dot
```

Comment:
I feel like selecting a cell adding an unread dot is a bug.

Reply:
i thought mail.app does this?

Reply:
I think it's a little confusing because the selection is selecting the cell in the demo app to mock an unread cell, not actually selecting or tapping the cell if that makes sense.

Reply:
Ah, that makes more sense. Thanks!